content/blog/command-provider-ga/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Announcing the general availability of the Command provider"
+
+date: 2024-06-20T11:02:20+02:00
+
+draft: false
```

Comment:
What does `draft` do?

Reply:
> # The draft setting determines whether a post is published. Set it to true if
> # you want to be able to merge the post without publishing it.

from archetypes/blog-post/index.md

I'm not 100% sure how to interpret this. I think we'll merge this PR once it's ready to publish, therefore I set it to true.